WooCommerce/Classes/ViewRelated/Orders/Order Creation/ProductsSection/AddProductToOrder.swift

```diff
@@ -0,0 +1,126 @@
+import SwiftUI
```

Comment:
Not sure if this is supposed to be part of the change? I don't remember seeing this during the first review....

Reply:
Oops, this was my mistake with the merge commit — accidentally re-added it instead of confirming it should be removed. I just fixed that in [66fa007](https://github.com/woocommerce/woocommerce-ios/pull/6685/commits/66fa00751cf38128ff3ae969088872d361dbd2fa).

Reply:
Thanks Rachel! Glad you pointed this out Jos 😅